cypress/e2e/Quotas/quotasPopup-UK.cy.js

```diff
@@ -0,0 +1,172 @@
+/* eslint-disable camelcase */
+/* eslint-disable max-len */
+const dayjs = require('dayjs');
+
+describe('🇬🇧 💡 | quotasPopup-UK | Verify quota dialogs |', function() {
+  const todaysDate = dayjs().format('DD MMM YYYY');
+  const previous_Date = dayjs().subtract(1, 'M').format('DD MMM YYYY');
+  const past_month = dayjs().subtract(0, 'M').month();
+  const past_date = dayjs().subtract(0, 'd').date();
+  const year = dayjs().subtract(0, 'y').year();
+  const future_Date = dayjs().add(7, 'day').format('DD MMM YYYY');
+  const future_date = dayjs().add(7, 'day').date();
+  const future_month = dayjs().add(1, 'M').month();
+  it('Quota Popup - Verify change the title to quota order number', function() {
+    cy.visit('/commodities/7306110000#quotas');
+    cy.contains('058039').click();
+    cy.get('#popup > div > div > article > h2').contains('Quota order number 058039');
+    cy.get('#popup > div > div > article > table > tbody > tr:nth-child(1) > th').should('not.contain', 'Quota order number');
+  });
+  it(`Quota Popup - Verify Balance as of ${todaysDate}`, function() {
+    cy.visit('/commodities/7306110000#quotas');
+    cy.contains('058039').click();
+    cy.get('#popup > div > div > article > h2').contains('Quota order number 058039');
+    cy.get('#popup > div > div > article > table > tbody > tr:nth-child(1) > th').contains(`Balance (as of ${todaysDate})`);
+    cy.should('not.contain', 'Current balance');
+  });
+  it('Quota Popup - Verify Comma separator in quota balance', function() {
+    cy.visit('/commodities/7306110000#quotas');
+    cy.contains('058039').click();
+    cy.get('#popup > div > div > article > table > tbody > tr:nth-child(1) > td').contains(',');
+    cy.get('#popup > div > div > article > table > tbody > tr:nth-child(2) > td').contains(',');
+  });
+  it('Quota Popup - Show pending balance', function() {
+    cy.visit('/commodities/7306110000#quotas');
+    cy.contains('058039').click();
+    cy.get('#popup > div > div > article > table > tbody > tr:nth-child(3)').contains('Kilogram (kg) remains available from the previous quota period. This will be transferred to the current quota period c. 20 working days after the end of the previous quota period.');
```

Comment:
```suggestion
    cy.get('#popup > div > div > article > table > tbody > tr:nth-child(3)').contains('Kilogram (kg) remains available from the previous quota period');
```

You don't need super long strings, here.

Validating exact content makes tests more brittle/doesn't actually add value